Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    platform: ""
+  client:
+    type: mobile app
+    name: Xiao Yu Zhou
+    version: "2.55.1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Mozilla/5.0 (iPhone; CPU iPhone OS 13_6 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) ; iPhone/13.6 XING/8.15.2 ttt_webview_iosm'
+  os:
+    name: iOS
+    version: "13.6"
+    platform: ""
+  client:
+    type: browser
+    name: Mobile Safari
+    version: ""
+    engine: WebKit
+    engine_version: "605.1.15"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Safari
+
+-
+  user_agent: 'Mozilla/5.0 (iPhone; CPU iPhone OS 16_4_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) XING-iPhone/23.10.2 (iOS; Light; de) iPhone/16.4.1 (iOS; Device) ttt_webview_iosm'
+  os:
+    name: iOS
+    version: "16.4.1"
+    platform: ""
+  client:
+    type: browser
+    name: Mobile Safari
+    version: ""
+    engine: WebKit
+    engine_version: "605.1.15"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Safari
+
+-
+  user_agent: 'Yandex.Music/2019-11-01.trunk.5885661'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Yandex Music
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Ya.Music/2022-10-06.trunk.10125214'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Yandex Music
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Ya.Music/2023-04-18.trunk.11307492'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Yandex Music
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'yapa/1.11.32 (Linux;Android 8.0.0) ExoPlayerLib/2.10.8'
+  os:
+    name: Android
+    version: "8.0.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: Yapa
+    version: "1.11.32"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'yapa/561 CFNetwork/1406.0.4 Darwin/22.4.0'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: Yapa
+    version: "561"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Zeno'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Zeno
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Zune/4.8'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Zune
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Libsyn4-peek'
```

Comment:
Is this really needed, if everything is essentially blank?
Based on my googling, this is either one of these 2 apps https://play.google.com/store/apps/details?id=com.libsyn.android.podcastBox21 and https://play.google.com/store/apps/details?id=com.libsyn.android.thefeed or this is a library distributed as described [here](https://libsyn.com/custom-mobile-apps/how-to-get-started/). The apps are 2.5.100 and 2.5.1 versions, so do not match the "4" in the agent, so most likely this is a library. Library assumption is also supported by the fact that [this page](https://libsyn.com/libsyn-5-features/) promoting "Libsyn 5" exists, and it also seems to include various things, include above mentioned apps.

Reply:
Libsyn is a podcast host, but I couldn't find an app for this. The one you found is probably it. I did find an extra ua looking into this, `Libsyn4-Download`.